examples/build.gradle

```diff
@@ -21,14 +21,14 @@ jar {
     }
 }
 
+configurations {
+    sparklingWaterAssemblyJar
+}
+
 dependencies {
-    // Sparkling Water Core
-    compile(project(":sparkling-water-core")) {
-        exclude group: "javax.servlet", module: "servlet-api"
-    }
 
-    // Sparkling Water ML
-    compile project(":sparkling-water-ml")
+    sparklingWaterAssemblyJar project(path: ':sparkling-water-assembly', configuration: 'shadow')
```

Comment:
Whole example should depend on the full assembly  as it has correct shadowing rules applied and and simulates how user is using sw